app/components/agenda/agendaitem/agendaitem-nav/template.hbs

```diff
@@ -1,16 +1,97 @@
-{{#each (if (is-pending activeTabs) defaultTabs (await activeTabs)) as |tab|}}
-  <li class="vl-tab"  data-test-agenda-agendaitem-tab={{tab.label}}>
-    <a href="" class="vl-tab__link
-      {{active-class activeAgendaItemSection tab.name class="active"}}"
-      {{action "setAgendaItemSection" tab.name}}>
-      {{t tab.label}}
-      {{#if (not-eq undefined tab.pillText)}}
-        <button class="vl-pill vl-pill--clickable" type="button">
-          {{await tab.pillText}}
```

Comment:
Hier heb je de selector aangepast, dit zorgt ervoor dat `  agendaItemDossierTab: '[data-test-agenda-agendaitem-tab="agendaitem-case"]',` niet meer gevonden wordt in de cypress testen (document-version.spec.js, meeting-actions.spec.js).

Pas je dit nog even aan? Daarna misschien best alle testen runnen om te elimineren dat het nog op andere plekken voorkomt.

Reply:
Test selector fixed ... full test run todo